webui/react/src/components/BatchActionConfirmModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { ExperimentAction } from '../types';
+
+import { Modal } from './kit/Modal';
+
+interface Props {
+  batchAction: ExperimentAction;
+  itemName?: string;
+  selectAll?: boolean;
+  danger?: boolean;
```

Comment:
nit: this should be removed since it's not being used

Reply:
good catch!